app/views/layouts/application.html.erb

```diff
@@ -28,10 +28,15 @@
     <% if @meta_section %>
       <meta name="govuk:section" content="<%= @meta_section %>">
     <% end %>
+    <%= cookieless_variant.analytics_meta_tag.html_safe %>
   </head>
 
   <body class="mainstream <%= yield :body_classes %>">
     <div id="wrapper" class="<%= wrapper_class(@publication || @presenter) %>">
+      <% unless cookieless_variant.variant?('Z') %>
+        <meta data-variant="<%= cookieless_variant.variant_name %>" data-module="track-variant">
```

Comment:
Is there a reason why this meta tag isn't in the head too?

Reply:
Good point - it makes me think actually that the previous analytics tag above is redundant. Let me check and get back to you

Reply:
So I've double checked and we do need both of the `meta` tags in this commit.

The first on line 31 is going to be picked up by the analytics code in Static and will be used to include this test as a custom dimension when any existing events are sent to GA. This is necessary when the existing hit event is sent to GA if users accept cookies, as we want to include the variant of this test that the user was in when they accepted.

The meta tag on line 37 is used by the track-variant.js file to get the variant of the test that the user is seeing. I included this as a separate meta tag specifically to be able to name the tag and easily get the value out - this isn't possible with the meta tab defined on line 31, which is of the form:

```
<meta name="govuk:ab-test" content="EducationNav:A" data-analytics-dimension="207" data-allowed-variants="A,B">
```